src/features/layers_panel/components/Layer/useControlElements.tsx

```diff
@@ -13,12 +13,18 @@ export function useControlElements(
     show: () => void;
     download: () => void;
   },
+  skipControls?: {
```

Comment:
I am more used to positive APIs than to negative ones, if that's just me you can dismiss this.